### DISCUSSION TITLE
How to use xsl transforms for Wix toolset's heat?

### DISCUSSION THREAD

**Discussion by kevfoerster** [cake-build/cake#3622] [Extension Q&A] [answered]
I am building an installer using the WiX toolset and Cake with the [appropriate extension](https://cakebuild.net/dsl/wix/). WiX has a harvester called [heat](https://wixtoolset.org/documentation/manual/v3/overview/heat.html). It has a switch `-t filename`, allowing to specify a xslt file with which to transform the original output. I could not find this switch in Cake's [HeatSettings](https://cakebuild.net/api/Cake.Common.Tools.WiX.Heat/HeatSettings/). However, I now need to transform the xml produced by heat.

What is the recommended way to do this?

**Comment by nils-a** [comment, marked as answer]
Have you tried using the `ArgumentCustomization` of the [HeatSettings](https://cakebuild.net/api/Cake.Common.Tools.WiX.Heat/HeatSettings/)? 
Something like `ArgumentCustomization = args=>args.Append("-t filename")`

**Reply by kevfoerster** [reply]
Thank you, that did it! Didn't see the property before :-).
